starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaDrmBridge.java

```diff
@@ -373,6 +373,37 @@ private MediaDrmBridge(String keySystem, UUID schemeUUID, long nativeMediaDrmBri
     mSchemeUUID = schemeUUID;
     mMediaDrm = new MediaDrm(schemeUUID);
 
+    // Get info of hdcp connection, as 4k drm require hdcp 2.2
+    switch (mMediaDrm.getConnectedHdcpLevel()) {
+      case MediaDrm.HDCP_V1:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_V1.");
+        break;
+      case MediaDrm.HDCP_V2:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_V2.");
+        break;
+      case MediaDrm.HDCP_V2_1:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_V2_1.");
+        break;
+      case MediaDrm.HDCP_V2_2:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_V2_2.");
+        break;
+      case MediaDrm.HDCP_V2_3:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_V2_3.");
+        break;
+      case MediaDrm.HDCP_NONE:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_NONE.");
+        break;
+      case MediaDrm.HDCP_NO_DIGITAL_OUTPUT:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_NO_DIGITAL_OUTPUT.");
+        break;
+      case MediaDrm.HDCP_LEVEL_UNKNOWN:
+        Log.i(TAG, "MediaDrm HDCP Level is HDCP_LEVEL_UNKNOWN.");
+        break;
+      default:
+        Log.i(TAG, String.format("MediaDrm HDCP Level is %d.", mMediaDrm.getConnectedHdcpLevel()));
```

Comment:
nit:
1. Use a variable at line 377, so we don't have to call the same function twice.
2. Rephrase the message to something like "Unknown MediaDrm HDCP level %d ...", i.e. explicitly mentioning unknown here.

Reply:
Fixed.